turbiniactl

```diff
@@ -37,6 +41,46 @@ from turbinia.pubsub import TurbiniaRequest
 log = logging.getLogger('turbinia')
 logger.setup()
 
+
+def PrintTaskStatus(project, region, days=1):
+  """Prints the recent history for Turbinia Tasks.
+
+  Args:
+    project (string): The name of the project.
+    region (string): The name of the zone to execute in.
+  """
+  function = GoogleCloudFunction(project_id=project, region=region)
+  start = datetime.now() - timedelta(days=days)
+  # Format this like '1990-01-01T00:00:00z' so we can cast it directly to a
+  # javascript Date() object in the cloud function.
+  start_string = start.strftime('%Y-%m-%dT%H:%M:%S')
+  response = function.ExecuteFunction(
+      'getrecenttasks', {'kind': 'TurbiniaTask', 'start':start_string})
+
+  if not response.has_key('result'):
+    log.error('Return from GCF does not contain a "result" key.')
+    sys.exit(1)
+
+  try:
+    results = json.loads(response['result'])
+  except ValueError as e:
+    log.error('Could not deserialize result from GCF: [{0!s}]'.format(e))
+    sys.exit(1)
+
+  task_results = results[0]
+  num_results = len(task_results)
+  if not num_results:
+    print 'No Tasks found.'
+    return
+
+  print '\nRetrieved {0:d} Task results:'.format(num_results)
+  for task in task_results:
+    success = "Successful" if task['successful'] else "Failed"
```

Comment:
Nit: use single quote to stay consistent

Reply:
Done